packages/fxa-auth-server/lib/routes/account.ts

```diff
@@ -609,7 +609,6 @@ export class AccountHandler {
         keysHaveChanged: true,
       }
     );
-    await this.db.resetAccountTokens(uid);
```

Comment:
I removed two `resetAccountTokens` calls because they both follow a `resetAccount` call, which already deletes those token records from the db.